docs/Articles/DownloadingTheXRTK.md

```diff
@@ -0,0 +1,66 @@
+# How to download the Mixed Reality Toolkit
+
+![](https://github.com/XRTK/XRTK-Core/raw/master/docs/logo.png)
+
+The XRTK provides many ways for users and developers to get access to the XRTK and the various extensions/platforms it provides.  These methods are tuned to use the common patterns most familiar to Unity developers, these include:
+
+* [Automatic UPM Install](https://github.com/XRTK/XRTK-Core/releases) - Provides the core package only, using UPM (Unity Package management) to deliver other modules
+
+* [Manual UPM install](#manual-upm-install) - Don't download anything, simply register the core XRTK project in your unity solution and let Unity download it for you
+
+* Offline Downloadable asset (tbc) - Provides the entire XRTK in an asset for use in offline scenarios
+
+* [GIT Submodules](#github-submodule) - For advanced developers, simply clone the project into your solution and update its submodules to get access to all platforms.
+
+In this article, we will walk through each approach to get you up and running, starting with the simplest first.
+
+## Automatic UPM Install
+
+Out preferred deployment approach is to fully utilize Unity's native package management solution to incorporate the XRTK in your solution, akin to the other modules Unity provides out of the box.
+This is the best / quickest and safest way to get XRTK in your solution and automatically updated.
+
+Just download the [XRTK-Core.unitypackage](https://github.com/XRTK/XRTK-Core/releases) asset and import it into your project to start building solutions straight away.  This adds a XRTK seed, which automatically registers the XRTK with the Unity package manager and starts its download and installation.
+
+For more instructions for how to get started, [click here](/docs/XRTKUPMInstall.md)
+
+## Manual UPM Install
+
+If you prefer to do things yourself, you can simply edit the ***manifest.json*** file in your Unity project to register the XRTK and begin the download and installation into your project.
+
+Simply add the following dependency into your manifest file, save and return to Unity to begin:
+
+```json
+{
+  "dependencies": {
+    "com.xrtk.core": "https://github.com/XRTK/XRTK-Core.git",
```

Comment:
These are missing the tag specifiers 

```
"com.xrtk.core": "https://github.com/XRTK/XRTK-Core.git#0.1.10",
```

Reply:
I found that if I didn't put a specifier on, it still for the latest, which should be preferable than specifying a version when manually editing.  Our recommendation should be to use the seed asset which does it for you.
Could possibly add another section to talk about specifying a version too?

Reply:
Only because I've had to setup the default branch as UPM but that will change once Unity fixes the Package Manager bugs I've reported.